3rdparty/expreval/vallist.h

```diff
@@ -16,55 +16,58 @@ namespace ExprEval
 {
 // Value list item
 //--------------------------------------------------------------------------
+template<class Value>
```

Comment:
wrong indent for L17-19? (should be same as L20)

Reply:
fixed.